apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -186,19 +186,32 @@ type ResourceFlavorReference string
 
 // ClusterQueueStatus defines the observed state of ClusterQueue
 type ClusterQueueStatus struct {
-	// flavorsUsage are the used quotas, by flavor, currently in use by the
+	// flavorsReservation are the reserved quotas, by flavor, currently in use by the
 	// workloads assigned to this ClusterQueue.
 	// +listType=map
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=16
 	// +optional
-	FlavorsUsage []FlavorUsage `json:"flavorsUsage"`
+	FlavorsReservation []FlavorUsage `json:"flavorsReservation"`
+
+	// flavorsUsage are the used quotas, by flavor, currently in use by the
+	// workloads admitted in this ClusterQueue.
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=16
+	// +optional
+	FlavorsUsage []FlavorUsage `json:"flavorUsage"`
```

Comment:
```suggestion
	FlavorsUsage []FlavorUsage `json:"flavorsUsage"`
```

We shouldn't break the existing API.

Reply:
Oops, missed it thanks.
Yes, please revert

Reply:
done